.github/workflows/style.yml

```diff
@@ -27,7 +27,7 @@ jobs:
 
     - name: Install dependencies
       run: |
-        pip install flake8 black pylint cmake-format
+        pip install -r dev-requirements.txt
```

Comment:
All these packages are defined in `dev-requirements.txt`. Decided to use that instead of double specifying the bounds. I think it makes sense to use `dev-requirements` here 🤷🏻